api/sheets/get_variable.md

```diff
@@ -1,111 +1,122 @@
 ---
 layout: default
-title: Statistical Variable Value
+title: Statistical Variable Values
 nav_order: 3
 parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the value of a statistical variable at a given place and time
+# Getting the Value of a Statistical Variable at a Given Place and Time
 
-Given a list of [DCIDs](/glossary.html) representing [Place](https://datacommons.org/browser/Place) objects, a [StatisticalVariable](https://datacommons.org/browser/StatisticalVariable), and optionally a date, get the measurements of the specified variable in the specified places with a date (if specified).
+The`=DCGET(dcids, variable, date)` formula returns the measurements of a specified [statistical variable](https://docs.datacommons.org/glossary.html#variable) at a given place and optional time based on a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html). A complete list of variables can be found in the [graph browser](https://datacommons.org/browser/StatisticalVariable).
 
-## General information about this formula
+> **Note**:
+> Be sure to follow the instructions for [Installing and Enabling the Sheets Add-On](/api/sheets/) before using this formula.
 
-**Formula**: `=DCGET(dcids, variable, date)`
+## Formula
 
-**Required arguments**:
-
-*   [`dcids`](/glossary.html): A list of `Place` nodes, identified by their DCIDs.
-
-*   `variable` - The [StatisticalVariable](/glossary.html) whose measurement is sought.
-
-**Optional arguments**:
+```
+=DCGET(dcids, variable, date)
+```
 
-*   `date` - The date or dates of interest. If this argument is not specified, the API will return the latest observation of the variable.
+### Required Arguments
 
-## Assembling the information you will need to use this formula
+* `dcids`: A list of [Place](https://datacommons.org/browser/Place) nodes, identified by their [DCIDs](/glossary.md#dcid).
```

Comment:
The DCID link is broken, switch to .html
```suggestion
* `dcids`: A list of [Place](https://datacommons.org/browser/Place) nodes, identified by their [DCIDs](/glossary.html#dcid).
```

Reply:
good catch!